src/lib/ctime/di_ops.c

```diff
@@ -279,3 +279,30 @@ select_array_member_cumulative_timei(const uint64_t *entries, int n_entries,
 
   return i_chosen;
 }
+
+/**
+ * If <b>s</b> is true, then copy <b>n</b> bytes from <b>src</d> to
+ * <b>dest</b>.  Otherwise leave <b>dest</b> alone.
+ *
+ * This function behaves the same as
+ *
+ *      if (s)
+ *         memcpy(dest, src, n);
+ *
+ * except that it tries to run in the same amount of time whether <b>s</b> is
+ * true or not.
+ **/
+void
+memcpy_if_true_timei(bool s, void *dest, const void *src, size_t n)
+{
```

Comment:
It the "timei" a typo or?

Reply:
I think it stands for "time invariant". Do you think the function name should be changed? I thought of changing it to `memcpy_if_true_timeinvariant` but not sure if that would be better. I can also add a comment that clarifies that `timei` stands for time invariant, but that seems a bit silly too. Any ideas?

Reply:
We already have gt_i64_timei(); I'm wondering if there is some other standard string we should use for this stuff.

Reply:
Ah great, didn't know what it meant. I'm fine with it.